users/vieting/experiments/librispeech/librispeech_100_ctc/fairseq_finetuning/ctc_standalone/serializer.py

```diff
@@ -38,6 +37,8 @@ def serialize_training(
     pytorch_train_step = Import(
         code_object_path=package + ".%s.train_step" % network_module, unhashed_package_root=PACKAGE
     )
+    # paths might have been set differently in the mean time
+    from .default_tools import I6_MODELS_REPO_PATH, FAIRSEQ_PATH
```

Comment:
Ah I see this is why you wanted to do it via the global variable. But can't you modify `get_training_config()` to pass the fairseq path to the serialization? I personally think it's cleaner not to always change the global variable, especially since the hash depends on it. It's easy to introduce strange behavior this way.

Reply:
Done with the newest commit :)